mobile/src/actions/index.js

```diff
@@ -49,7 +50,7 @@ export const registerDevice = (router, location) => {
         clientParams: {
           redirectURI: 'http://localhost',
           softwareID: 'io.cozy.mobile.files',
-          clientName: 'Mobile Cozy Files',
+          clientName: genId(),
```

Comment:
> Hi, @enguerran would you like to let 3M (http://localhost)` access to your files ?

May be something along the lines of `CozyFiles Application on Device' + cordova.device.model + genID()` would be clearer.